### DISCUSSION TITLE
Show playlist in UI

### DISCUSSION THREAD

**Discussion by tecbeat** [music-assistant/support#1863] [Feature requests and ideas]
I was using a quick selection of my playlists in the dashboard before I switched to Musik assistant. And so now I need a list of all playlists to be able to display them. Unfortunately, I have not found this as a service/state. Is there a way to access this?

**Comment by OzGav** [comment]
Please review the FAQs https://github.com/orgs/music-assistant/discussions/710#discussioncomment-5441452

**Reply by tecbeat** [reply]
Sorry for the inaccuracy, I meant access directly in Home Assistant without the detour via the Music Assistant frontend. So that I can create a list of playlists and start the music with one click directly from the standard home assistant dashboard.

**Reply by OzGav** [reply]
Depends. You could use the `mass.search` service call to get a list of playlists. However, if you are trying to list your personal playlists only then I think you might have to hardcode them in an `input_select`

**Reply by tecbeat** [reply]
Ok thanks. Maybe this is an improvement for the future.

**Reply by OzGav** [reply]
I have changed the category of this to a feature request. We will see how any other people would like this.

**Comment by OzGav** [comment]
There is a new action `mass.get_library` which will allow you to get a list of all your playlists

**Reply by userMak** [reply]
thanks!

**Reply by tecbeat** [reply]
thx
